lib/modules/datasource/azure-bicep-resource/schema.ts

```diff
@@ -2,37 +2,35 @@ import { z } from 'zod';
 
 export const BicepResourceVersionIndex = z
   .object({
-    Resources: z.record(
+    resources: z.record(
       z.string(),
       z.object({
-        RelativePath: z.string(),
-        Index: z.number(),
+        $ref: z.string(),
       }),
     ),
-    Functions: z.record(
+    resourceFunctions: z.record(
       z.string(),
       z.record(
         z.string(),
         z.array(
```

Comment:
This array should be replaced with `z.unknown()`, if we're going to ignore it and only use the keys from upper-level record.